### ISSUE TITLE
AR/OD/etc. changer consistency

### ISSUE THREAD

**Issue by FenekAlfa** [itdelatrisu/opsu#143]
I noticed that some mods(like DT) affect the custom AR/OD/etc, while others(like HR) don't. It would be nice to have a consistent behaviour.

If you choose to make all of them affect custom values, it'd be nice to show the values after the mods somewhere. On the other hand, if you make them not affect the custom overrides, you'd need to extend the range from -5 to 11(maybe more for OD, I'm not sure) to not lose any values.

**Comment by itdelatrisu** [comment]
I thought that all of the values were overridden?  Here's the relevant code:
```
		// map-based properties, re-initialized each game
		float multiplier = GameMod.getDifficultyMultiplier();
		float circleSize = Math.min(beatmap.circleSize * multiplier, 10f);
		float approachRate = Math.min(beatmap.approachRate * multiplier, 10f);
		float overallDifficulty = Math.min(beatmap.overallDifficulty * multiplier, 10f);
		float HPDrainRate = Math.min(beatmap.HPDrainRate * multiplier, 10f);

		// fixed difficulty overrides
		if (Options.getFixedCS() > 0f)
			circleSize = Options.getFixedCS();
		if (Options.getFixedAR() > 0f)
			approachRate = Options.getFixedAR();
		if (Options.getFixedOD() > 0f)
			overallDifficulty = Options.getFixedOD();
		if (Options.getFixedHP() > 0f)
			HPDrainRate = Options.getFixedHP();
```
And all the values in osu! are capped at 10, too, aren't they?

**Comment by FenekAlfa** [comment]
https://osu.ppy.sh/s/68740
I used this map to compare DT and non-DT accurately(some of the dificulties are slowed down and play at normal speed with DT). Custom AR was set to 8,5 and I used Relax. Both screenshots show the moment circle number 6 appears.

http://i.imgur.com/s8nTXex.png
DT

http://i.imgur.com/Q10drjN.png
No DT

http://w.ppy.sh/6/68/ARTable.png
Also every mod has it's own value cap for each value. Here's the table for AR(that's where my -5 to 11 guess comes from), and if I find info for OD, CS and HP I'll link them as well.
